courses/free-courses-en.md

```diff
@@ -909,6 +909,7 @@
 * [Learn Python Tutorial](https://www.kaggle.com/learn/python) (Kaggle)
 * [Learn to program in Python](https://www.codecademy.com/learn/python)
 * [Learn to Program: The Fundamentals](https://www.coursera.org/learn/learn-to-program) (Coursera)
+* [NOC:The Joy of Computing using Python, IIT Ropar](https://nptel.ac.in/courses/106106182) - Prof. Sudarshan Iyengar (NPTEL)
```

Comment:
1. We do not list honorifics such as 'prof.'
2. This course appears to have and end date. Because we'd have to remove it then, we won't add it now.

Reply:
1. The end date is only for those who wish to enroll for the certification. The videos are available in YouTube throughout.
2. The certification runs twice every year, unless archived. So the end date is for the current session.
3. Even archived, the videos remain accessible.